app/controllers/welcome_controller.rb

```diff
@@ -28,6 +29,20 @@ def mentor
     @report = params[:report].blank? ? "basics" : ERB::Util.html_escape(params[:report])
   end
 
+  def participant
+    @participant = @current_user.person
+    @participant.type = "Participant"
+    @participant.save
```

Comment:
Ahh. I was trying to debug here and forgot to change it before pushing. Yes, it is a very bad idea!